### PR TITLE
Some message box cleanup + DeveloperView Functionalites 

### PR DESCRIPTION
**Pull request by muazzamaqeel**

### CODE REVIEW

**Comment by muazzamaqeel** [comment]
@VincentStriegel This is just an intermediate progress, i will polish and fix the other things, please verify if everything is in place and didn't effect something else 

Thank you so much like always!!!!

File: Source Code/Front-End/UI (WPF)/Smart Pacifier - Tool/Smart Pacifier - Tool/Tabs/DeveloperTab/DeveloperView.xaml [hunk 1]
```diff
@@ -131,6 +131,12 @@
                             </GridViewColumn.CellTemplate>
                         </GridViewColumn>
 
+
```
Comment (VincentStriegel) [inline Source Code/Front-End/UI (WPF)/Smart Pacifier - Tool/Smart Pacifier - Tool/Tabs/DeveloperTab/DeveloperView.xaml:134]: Would it be possible to have dynamic headers?

**Comment by VincentStriegel** [comment]
Good work! I've test both local and server based DBs, both work.

* One thing I would add is a filter reset
* What is the purpose of the Apply Button? The filters applied automatically for me. 
* I was not able edit data -- see img below: 
Screenshot 2024-11-13 102238

File: Source Code/Front-End/UI (WPF)/Smart Pacifier - Tool/Smart Pacifier - Tool/Tabs/DeveloperTab/DeveloperView.xaml (hunk 1 above)
Comment (muazzamaqeel) [reply Source Code/Front-End/UI (WPF)/Smart Pacifier - Tool/Smart Pacifier - Tool/Tabs/DeveloperTab/DeveloperView.xaml:134]: Theoretically yes, thats a nice idea!
I will mark this down for the next improvement

**Comment by muazzamaqeel** [comment]
> Good work! I've test both local and server based DBs, both work.
> 
> * One thing I would add is a filter reset
> * What is the purpose of the Apply Button? The filters applied automatically for me.
> * I was not able edit data -- see img below:
>   Screenshot 2024-11-13 102238

Yes i think we need to remove that Apply Button as it was just an initial idea of the wireframe and the edit functionality i will finish today 

So then i will merge this branch for now, then will work on all the improvements that you have suggested as soon as possible

Thanks a lot for your detailed review

**Pull request merged**